### DISCUSSION TITLE
Free Laravel Hosting

### DISCUSSION THREAD

**Discussion by Yasbarros** [community/community#121159] [Programming Help] [answered]
### Body

I'm looking for hosting for my laravel project with mysql that is free, as it is only for viewing for a few days so hosting with a free time limit will do. Could anyone recommend one?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by fdistasio** [comment, marked as answer]
You could use the free tier of AWS or Google Cloud Platform:
-https://aws.amazon.com/it/free/webapps
-https://cloud.google.com/gcp

**Comment by josetorronteras** [comment]
Hi, at the time I tried https://render.com and https://fly.io.
I hope it works for you

**Comment by ebndev** [comment]
Hi @Yasbarros! Thanks for posting in the Community.

Let us know if any of the sites above worked for you or if you ended up going with something else. It would be great to reference this post in the future if other folks have the same issue 😁

**Comment by leminhluan4244** [comment]
These are some pages that I or my colleagues have found useful:

https://interdata.vn/free-hosting
https://cpanel.net/products/trial/
https://flashpanel.io/#pricing

I hope it works for you

**Reply by CopperEagle** [reply]
Hey @leminhluan4244, thanks for sharing. I did not know about these!
